mgmtsystem/scvmm.py

```diff
@@ -148,6 +148,12 @@ def list_vm(self, **kwargs):
             "Select name | ConvertTo-Xml -as String")
         return etree.parse(StringIO(data)).getroot().xpath("./Object/Property[@Name='Name']/text()")
 
+    def list_hosts(self, **kwargs):
+        data = self.run_script(
+            "Get-SCVMHost -VMMServer $scvmm_server |"
+            "Select Name | Where-Object {$_.Name -like 'qeblade*'}| ConvertTo-Xml -as String")
```

Comment:
It's fine, we can "pass" it in, but we need ti as an argument and not hardcoded in here